src/managedwriter/stream_connection.ts

```diff
@@ -357,7 +357,11 @@ export class StreamConnection extends EventEmitter {
    * Re open appendRows BiDi gRPC connection.
    */
   reconnect() {
-    this.trace('reconnect called');
+    this.trace(
+      'reconnect called with',
```

Comment:
nit: spacing here looks odd, unless node concat adds spaces between fragments.

Reply:
`console.log` adds spaces, but users can provide other loggers, so I'll use template strings here and add the spaces